ssg/constants.py

```diff
@@ -173,6 +173,15 @@
     "chromium": [
         "cpe:/a:google:chromium-browser",
     ],
+    "centos6": [
+        "cpe:/o:centos:centos:6",
+    ],
+    "centos7": [
+        "cpe:/o:centos:centos:7",
+    ],
+    "centos8": [
+        "cpe:/o:centos:centos:8",
+    ],
```

Comment:
all cpes need to be dropped from constants.py

Reply:
Actually a lot of CPE mappings can be dropped from `ssg/constants.py`.
I'm not sure if it would better be done in this PR or a subsequent one.
I lean towards doing it in a separate PR.

Reply:
sounds good

Reply:
`PRODUCT_TO_CPE_MAPPING` was removed